python/paddle/fluid/tests/unittests/test_dist_base.py

```diff
@@ -444,6 +447,46 @@ def _get_data(batch):
                 model.clear_gradients()
         print_to_out(out_losses)
 
+    def run_trainer_with_spawn(self, args):
+        # 1. enable dygraph
+        paddle.disable_static()
+
+        # 2. init seed
+        seed = 90
+        paddle.static.default_startup_program().random_seed = seed
+        paddle.static.default_main_program().random_seed = seed
+        np.random.seed(seed)
+        random.seed = seed
+
+        # 3. init parallel env
+        if args.update_method == "nccl2":
```

Comment:
As lilong12 has been creating a global parallel environment initialization entry in API 2.0 task, supporting both nccl and gloo, I think you should have a discussion with him.